python/cudf/cudf/core/column/lists.py

```diff
@@ -108,6 +111,58 @@ def set_base_data(self, value):
         else:
             super().set_base_data(value)
 
+    def serialize(self):
+        header = {}
+        frames = []
+        header["type-serialized"] = pickle.dumps(type(self))
+        header["dtype"] = pickle.dumps(self.dtype)
+        header["null_count"] = self.null_count
+        header["size"] = pickle.dumps(self.size)
```

Comment:
Why can we pass the null_count but need to pickle the size? Both should be integers.

Reply:
Great point... I was using [the string column methods](https://github.com/rapidsai/cudf/blob/96ee0518345009e14cb096026cf479bad9842fc4/python/cudf/cudf/core/column/string.py#L4783) as a reference, and the size was pickled there.  You are right that the size is just an integer, so I guess pickle shouldn't be necessary.